src/components/Table/head/selectableHeader.js

```diff
@@ -18,8 +18,8 @@ export default function SelectableHeader(props) {
     const name = `${tableId}-options`;
     const isDisabled = maxRowSelection === 0;
     const isRadio = maxRowSelection === 1;
-    const isIndeterminate = bulkSelection === 'some';
-    const isAllSelected = bulkSelection === 'all';
+    const bulkStateMap = new Map([['all', true], ['some', 'indeterminate'], ['none', false]]);
+    const checked = bulkStateMap.get(bulkSelection);
```

Comment:
good job
only put bulkStateMap outside the component since it has fixed values that do not depend on any prop or state because here the map is recreating in each re-render without need it.
Also, you can simply use an object:
const bulkStateMap = {
    all: true,
    some: 'indeterminate',
    none: false,
};